CONTRIBUTING.md

```diff
@@ -14,6 +14,61 @@ pull request.
 See [GitHub's official documentation](https://help.github.com/articles/using-pull-requests/)
 for more details.
 
+## Developer Setup
+
+Before making changes to markdown files in the handbook, developers should set
+up a [MkDocs development environment](https://www.mkdocs.org/#installation).
+The most consistent way is manually, through Python:
+
+1. Download and [install Python](https://www.python.org/)
+   (3.8 or later is recommended)
+2. (Optional) Create a virtual environment named `mkdocs` in the current
+   directory: `python -m venv mkdocs`
+3. (Optional) Activate the virtual environment by running `activate` in the
+   `Scripts` sub-directory of the `mkdocs` virtual environment
```

Comment:
Hmm, this was my convoluted way to avoid using Windows- or Linux-specific slashes to represent file paths like `mkdocs\Scripts\activate`. How about:

"To activate the newly created `mkdocs` virtual environment, switch to the `Scripts` sub-directory and run `activate`"?